etc/wordpress.json

```diff
@@ -0,0 +1,20 @@
+{
+  "consul": "consul:8500",
+  "services": [
+    {
+      "name": "wordpress",
+      "port": 80,
+      "publicIp": true,
+      "health": "/usr/bin/curl --fail -s http://localhost/health",
```

Comment:
I'm thinking to use `wp option get site url` as the health test here. That will confirm the WP config is working, but won't test Apache at all. So, it has downsides. That said, I can run both tests.